docs/UserGuide.md

```diff
@@ -3,436 +3,713 @@ layout: page
-| **Find a Customer by Details** | `filter <FLAG>/ <FLAG FIELD>`                                                                                                                                  | `filter n/ TAN LESHEW`                                                                                         |
-| **Help**                       | `help`                                                                                                                                                         | `help`                                                                                                         |
-| **Exit**                       | `exit`                                                                                                                                                         | `exit`                                                                                                         |
+## 8. Command Summary
+
+| **Action**                 | **Command Format**                                                                                                           | **Example**                                                                                               |
+|----------------------------|------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| **Add New Client**         | `add n/<NAME> p/<PHONE> e/<EMAIL> a/<ADDRESS> j/<JOB> i/<INCOME> [t/<TIER>] [rn/<REMARK>]`                                   | `add n/ GORDON MOORE p/ 99007766 e/ gmoore@ntu.sg a/ COM3 j/ engineer i/ 99999 t/ gold rn/ remark`        |
+| **Delete Existing Client** | `delete <INDEX>`                                                                                                             | `delete 69`                                                                                               |
+| **Edit Existing Client**   | `edit <INDEX> n/<NAME> p/<PHONE> e/<EMAIL> a/<ADDRESS> j/<JOB> i/<INCOME> [t/<TIER>] [rn/<NEW REMARK>] [ra/<APPEND REMARK>]` | `edit 69 n/ GORDON MOORE p/ 77337733 e/ gmoore_new@ntu.sg a/ COM3 j/ doctor i/ 1000000000 ra/ added info` |
+| **List All Clients**       | `list`                                                                                                                       | `list`                                                                                                    |
+| **Filter Client List**     | `filter n/<NAME> p/<PHONE> e/<EMAIL> a/<ADDRESS> j/<JOB> r/<REMARK> t/<TIER> i/<INCOME>`                                     | `filter n/ GORDON MOORE j/ doctor t/ gold`                                                                |
```

Comment:
Perhaps you can use the optional parameters format for the flags in `filter` command.

Reply:
Great idea, I've included this change.